exporters/trace/ocagent/README.md

```diff
@@ -26,6 +26,12 @@ For Maven add to your `pom.xml`:
     <version>0.21.0</version>
     <scope>runtime</scope>
   </dependency>
+  <dependency>
+    <groupId>io.netty</groupId>
+    <artifactId>netty-tcnative-boringssl-static</artifactId>
```

Comment:
Very confused about this:
https://github.com/census-instrumentation/opencensus-java/blob/master/examples/pom.xml#L102
https://github.com/census-instrumentation/opencensus-java/blob/master/examples/pom.xml#L128
and here with 2.0.20.Final version??

Can we clarify what exactly it is needed? @zhangkun83 can you help us with this?

Reply:
IIRC, previously `io.grpc:grpc-netty-shaded` shaded the `netty-tcnative-boringssl` artifact, but at some point `netty-tcnative-boringssl` was removed from `grpc-netty-shaded`, and as a result we had to add it explicitly as a runtime dependency. See https://github.com/census-instrumentation/opencensus-java/pull/1247. I believe this runtime dependency is still needed but I'll defer to @zhangkun83 for clarification.

Reply:
@ejona86 should be able to help.

Reply:
`netty-tcnative-boringssl-static` is not necessary, and specifying it should be useless. The shaded netty can't use it. grpc-netty-shaded _must_ include tcnative for it to work at all.

#1247 doesn't really provide any information for me to correct or verify. So I can only say, "no, that's not right." Is there any more information as to why this was believed? I'm surprised a bug wasn't opened and that we weren't notified, especially since there was no mention of dependency changes in the v1.10 the release notes.

FYI: I'm not part of the census-instrumentation github org, so I can't route github email for notifications like this. They go to my person email address and tend to be lost.

Reply:
Taking a look at #1897, the error message implies you aren't using `grpc-netty-shaded`. It looks like [`io.opencensus:opencensus-exporter-trace-ocagent` depends on grpc-netty](https://search.maven.org/artifact/io.opencensus/opencensus-exporter-trace-ocagent/0.22.0/jar). grpc-netty-shaded is nowhere to be found. Looking at #1247 again, it is clear it was using grpc-netty and not grpc-netty-shaded. So now the mystery is solved.

netty-tcnative-boringssl-static is necessary when using grpc-netty instead of grpc-netty-shaded. To choose the correct version of tcnative, you can either follow the release notes or look at the table under "Below are known to work version combinations:" at https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty

Reply:
Thanks for providing the context, it's very useful. Yes we need to depend on `grpc-netty` rather than `grpc-netty-shaded` in these two artifacts because they used `io.netty.handler.ssl.SslContext`. 

Currently we depend on grpc 1.19.0, from the table I think `tcnative:2.0.20.Final` should work.